src/vue/backend/src/templates/GeneralSettings.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <idx-block className="general-settings">
+        <idx-form-group>
+            <idx-form-label for="website-wrapper"><strong>Name Your Global Website Wrapper</strong><br> Wrappers set the overall styling of your IDX Broker pages, some words about maintaining a consistent design between WordPress and IDX Broker.</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="website-wrapper"
+                :value="wrapperName"
+                @keyup="wrapperChangeState($event)"
+            />
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-label"><strong>Google reCAPTCHA</strong><br> Google reCAPTCHA v3 helps to prevent spammers from filling out your forms.</idx-block>
+            <idx-block className="control-toggle-slider">
+                {{ toggleLabel }}
+                <idx-toggle-slider
+                    uncheckedState="No"
+                    checkedState="Yes"
+                    @toggle="generalSettingsStateChange({ key: 'reCAPTCHA', value: !reCAPTCHA })"
+                    :active="reCAPTCHA"
+                    :label="toggleLabel"
+                ></idx-toggle-slider>
+            </idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-form-label for="update-frequency"><strong>Update Frequency</strong><br> Choose how often IMPress gets updates from your IDX Broker account.</idx-form-label>
+            <idx-custom-select
+                id="update-frequency"
```

Comment:
Custom select already has a label in the component, can you run this through with a screen reader and verify there isn't repetition?